.gitpod.yml

```diff
@@ -0,0 +1,41 @@
+image:
+  file: .gitpod.Dockerfile
+ports:
+- port: 27017 # mongodb
+  onOpen: ignore
+- port: 8000 # client
+  onOpen: open-preview
+- port: 9228 # node debug
+  onOpen: ignore
+- port: 3000 # api
+  onOpen: ignore
+- port: 9229 # node debug
+  onOpen: ignore
+
+tasks:
+- name: db
+  # starting mongo in background, so it doesn't block prebuilds
+  before: >
```

Comment:
Theoretically, can this `before` section be completed after starting the `init` server section? If so, let's place this to the server `before` section.

Reply:
yes, in theory. Will update

Reply:
Can we await port 27017 in server `before` section?

Reply:
Yes, that's possible as well. That should go before `npm run seed`, no?
Should I then keep an extra terminal for mongo?

Reply:
> Yes, that's possible as well. That should go before `npm run seed`, no?

Yes, and before `npm run develop:server`